### PR TITLE
Merge 1.4 into 1.5

### PR DESCRIPTION
**Pull request by aRn0D**

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

File: CHANGELOG-1.4.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-- PIM-5820: Fix product value filtering by channel and locale in structured product normalizer
-- PIM-5687: Fix an issue that prevented the removal of a product from a variant group in MongoDB
 - PIM-5863: Fix an issue that prevented the association of multiple products at once in the association tab
 - PIM-5389: Fix an issue that would render the history panel multiple times
 
+# 1.4.25 (2016-06-03)
+
+## Bug fixes
+- PIM-5820: Fix product value filtering by channel and locale in structured product normalizer
+- PIM-5687: Fix an issue that prevented the removal of a product from a variant group in MongoDB
```
Comment (phaseinducer) [inline CHANGELOG-1.4.md]: This diff is weird no?

Comment (aRn0D) [inline CHANGELOG-1.4.md]: Yes I don't know what we did with Samir before but it was wrong!

**Comment by phaseinducer** [comment]
Ok, GTM =)

**Pull request merged**